modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/configs/v1/ConfigSuccessTest.java

```diff
@@ -97,7 +97,7 @@ public void testGetAuthenticator() throws Exception {
                 .statusCode(HttpStatus.SC_OK)
                 .body("id", equalTo(SAMPLE_AUTHENTICATOR_ID))
                 .body("name", equalTo("BasicAuthenticator"))
-                .body("displayName", equalTo("basic"))
+                .body("displayName", equalTo("Username & Password"))
```

Comment:
are we changing the api response now? There can be people using this API and if we are doing these kind of changes, we need to document that